pkg/util/template.go

```diff
@@ -8,17 +8,23 @@ import (
 	"net/url"
 	"os"
 	"path/filepath"
+	"reflect"
 	"strings"
 	"text/template"
 
 	"github.com/Masterminds/sprig/v3"
 	"github.com/ghodss/yaml"
 )
 
-var extraTemplateFuncs = template.FuncMap{
-	"urlEncode": url.QueryEscape,
-	"toYaml":    toYaml,
-}
+var (
+	extraTemplateFuncs = template.FuncMap{
+		"lookup":    lookup,
+		"toYaml":    toYaml,
+		"urlEncode": url.QueryEscape,
+	}
+
+	zeroValue = reflect.Value{}
```

Comment:
Using `reflect.Value.IsValid` is usually the preferred way to check I believe.

Reply:
Oh, nice. Fixed.